### PR TITLE
Fix internal 72

### PR DESCRIPTION
**Pull request by jannesblobel**
Add group identifier to IDE and CLI

### CODE REVIEW

File: source-code/ide-extension/src/services/telemetry/implementation.ts [hunk 1]
```diff
@@ -38,6 +38,15 @@ async function capture(args: CaptureEventArguments) {
...
 	}
+	if (args.event === "IDE-EXTENSION activated") {
+		telemetry.groupIdentify({
+			groupType: "repository",
+			groupKey: gitOrigin,
+			properties: {
+				name: gitOrigin,
+			},
+		})
+	}
```
Comment (samuelstroschein) [inline source-code/ide-extension/src/services/telemetry/implementation.ts:49]: Same here. I would implement this directly after https://github.com/inlang/inlang/blob/84116139da02753ba3c69864ae439a3e43fc84e6/source-code/ide-extension/src/main.ts#L20-L27

**Review by samuelstroschein — CHANGES_REQUESTED** [review]
Furthermore, don't forget to create a changeset with `npx changeset`. This should be a patch note

File: source-code/ide-extension/src/services/telemetry/implementation.ts (hunk 1 above)
Comment (jannesblobel) [reply source-code/ide-extension/src/services/telemetry/implementation.ts:49]: That was my first approach, but I had to import GitOrigin in this case. I assume that it is not so pretty to import the GitOrign here. 
But i will look into it again :)

Comment (samuelstroschein) [reply source-code/ide-extension/src/services/telemetry/implementation.ts:49]: Ah fair. Looks cleaner regardless i would assume :)

**Review by samuelstroschein — APPROVED** [review]

**Comment by samuelstroschein** [comment]
@jannesblobel good to merge. just run `npx changeset` and create a changeset please

**Comment by jannesblobel** [comment]
@samuelstroschein  nur nochmal kurz um sicher zu gehen, dass das changeset auch richtig ist. 
habe auch noch ganz kurz den groupIdentify in die badge gepackt  nicht unbedingt notwendig aber sollte auch nicht schaden

**Comment by samuelstroschein** [comment]
@jannesblobel sieht richtig aus (patch note)

**Pull request merged**